src/gsy_e/gsy_e_core/sim_results/endpoint_buffer.py

```diff
@@ -57,11 +57,17 @@ def __init__(self, is_scm: bool):
         else:
             self.simulation_raw_data_validator = get_schema_validator(
                 "simulation_raw_data")
+        self.simulation_configuration_tree_validator = get_schema_validator(
+            "results_configuration_tree")
 
     def validate_simulation_raw_data(self, data: Dict):
         """Validate flattened_area_core_stats_dict."""
         self.simulation_raw_data_validator.validate(data=data, raise_exception=True)
 
+    def validate_configuration_tree(self, data: Dict):
+        """Validate configuration_tree dict."""
+        self.simulation_configuration_tree_validator.validate(data=data, raise_exception=False)
```

Comment:
is there any reason you decided not to raise exceptions here? We are at the moment not raising any exceptions in the `validate` implementation, so you can set `raise_exception=True` without any worries.

Reply:
Good point, let me bring it back to True. Thanks!